pkg/network/http/monitor.go

```diff
@@ -47,15 +44,15 @@ func NewMonitor(c *config.Config) (*Monitor, error) {
 		return nil, fmt.Errorf("error setting up http ebpf program: %s", err)
 	}
 
+	if err := mgr.Init(); err != nil {
```

Comment:
Just curious about this -- was this in the wrong place before? (i.e. after `GetProbe`)

Reply:
not really, I was just trying to re-organize the code :)

Reply:
Hm.. so calling a function on `mgr` before calling `Init` is ok?

Reply:
it depends. you can call `GetProbe`, for example, but not `GetMap`.
What `Manager.Init()` does on a very high level is essentially *loading* the eBPF programs, while `Start()` takes care of *attaching* them.